lib/wp2ghost.js

```diff
@@ -119,10 +119,18 @@ exports.fromStream = function(stream) {
 
       var date;
 
+      date = item['wp:post_date_gmt'];
+      if (date == null) {
+          date = item['wp:post_date'];
+        if (date == null) {
+          date='0000-00-00 00:00:00'
+        }
+      }
+
       if (item['wp:post_date_gmt'] !== "0000-00-00 00:00:00") {
```

Comment:
Well it depends on if you want to check for if `post_date `is not defined as well.